### ISSUE TITLE
Error [INTERACTION_ALREADY_REPLIED]: The reply to this interaction has already been sent or deferred.

### ISSUE THREAD

**Issue by carabax** [CarelessInternet/discord.js-games#185]
After using the command and waiting for the game to finish, the error `[INTERACTION_ALREADY_REPLIED]: The reply to this interaction has already been sent or deferred.` appears.

![2022-07-23_15-14](https://user-images.githubusercontent.com/100368550/180598905-6e3d56f5-75fc-491e-9335-8a1476ada692.png)

**My code:**
```
import {SlashCommandBuilder} from '@discordjs/builders';
import {MessageEmbed, Permissions, Client, Intents} from 'discord.js';

const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MEMBERS, Intents.FLAGS.GUILD_PRESENCES] });

import { blackjack } from 'discord.js-games';

export default {
	data: new SlashCommandBuilder()
	.setName('blackjack')
	.setDescription('Blackjack command'),
	restriction: FLAGS.CHANNEL,
	timeout: [-1, FLAGS.USER],
	permissions: [],
	async execute(interaction) {

		const result = await blackjack({ message: interaction });

		console.log('user won', result);

	}
};
```

**Comment by CarelessInternet** [comment]
Can't reproduce the error with the given code. The screenshot you provided suggests that nothing wrong is coming from the library, since I don't see any trace to discord.js-games. Have you checked your interactionCreate.js file to make sure a duplicate interaction reply doesn't occur?

**Comment by carabax** [comment]
Nope, my `interactionCreate.js` is working fine, the error occurs only after using the library

**Comment by CarelessInternet** [comment]
Can you give more information on how to reproduce the error? E.g. what buttons you clicked and how long you waited before making a move.

**Comment by CarelessInternet** [comment]
Closing this issue due to no response, feel free to reopen if you are still experiencing the problem and will give more info about how to reproduce the error.
